src/topology/algebra/valuation.lean

```diff
@@ -13,29 +13,27 @@ import ring_theory.valuation.basic
 
 In this file, we define the non archimedean topology induced by a valuation on a ring.
 The main definition is a `valued` type class which equips a ring with a valuation taking
-values in a group with zero (living in the same universe). Other instances are then deduced from
-this.
+values in a group with zero. Other instances are then deduced from this.
 -/
 
 open_locale classical topological_space
 open set valuation
 noncomputable theory
 
-universe u
+universes v u
 
 /-- A valued ring is a ring that comes equipped with a distinguished valuation.-/
-class valued (R : Type u) [ring R] :=
-(Γ₀ : Type u)
-[grp : linear_ordered_comm_group_with_zero Γ₀]
+class valued (R : Type u) [ring R] (Γ₀ : out_param (Type v))
+  [linear_ordered_comm_group_with_zero Γ₀] :=
 (v : valuation R Γ₀)
 
-attribute [instance] valued.grp
-
 namespace valued
-variables {R : Type*} [ring R] [valued R]
+variables {R : Type u} [ring R] (Γ₀ : out_param (Type v)) [linear_ordered_comm_group_with_zero Γ₀]
```

Comment:
```suggestion
variables {R : Type u} [ring R] (Γ₀ : Type v) [linear_ordered_comm_group_with_zero Γ₀]
```
I think `out_param` only means something in the `class` definition above, but maybe @Vierkantor  can clarify

Reply:
That's exactly right: `out_param` is only checked in the parameters of a class, before starting instance search.

Reply:
Ok, thanks!